frontend/src/app/modules/status/pages/status.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="mx-auto lg:container px-4 py-4 sm:px-8
+            grid grid-cols-1 md:grid-cols-3" *transloco="let t">
```

Comment:
Add some spacing between when vertically aligned and make sure they cover the entire width
```suggestion
            grid grid-cols-1 md:grid-cols-3 gap-2 w-full" *transloco="let t">
```